### PR TITLE
Update navigation

### PR DESCRIPTION
**Pull request by roll**
Hi @lwinfree,

Please take a look. 

I just updated the navigation (mostly tooling) to make it consistent with what we have at the moment.

Demo: https://fd-delta.vercel.app

### CODE REVIEW

**Review by lwinfree — COMMENTED** [review]
Hi @roll this is great! I have 2 suggested changes to the wording:

- on the file `site/tooling/libraries/README.md`, I suggest rewording the tagline (line 3) as: "The Frictionless code is available in 10 languages". I think the term "low-level drivers" is confusing.

- on the file `site/tooling/application/README.md` I suggest changing line 11 `Helper Tools` to `Browser Tools`

**Comment by roll** [comment]
Thanks @lwinfree, fixed :+1:
